Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java

```diff
@@ -21,11 +22,17 @@
 
 public final class MenuFragment extends Fragment implements View.OnClickListener
 {
+  private View mPauseEmulationTv;
+  private View mUnpauseEmulationTv;
```

Comment:
These two don't need to have `Tv` in the name anymore. (Feel free to ignore this if you want to.)